builder/bundles.go

```diff
@@ -877,7 +878,40 @@ func createVersionsFile(baseDir string, packagerCmd []string) error {
 	return w.Flush()
 }
 
-func fixOSRelease(filename, version string) error {
+func fixOSRelease(b *Builder, filename, version string) error {
+
+	//Replace the default os-release file if customized os-release file found
+	path := b.Config.Mixer.OSReleasePath
+	if path != "" {
+
+		// Validate file path
+		_, err := os.Stat(path)
+		if err != nil {
+			return err
+		}
+
+		fSrc, err := os.OpenFile(path, os.O_RDONLY, 0666)
+		if err != nil {
+			return err
+		}
+
+		fDst, err := os.OpenFile(filename, os.O_RDWR, 0666)
+		if err != nil {
+			return err
+		}
```

Comment:
Same as above, use `defer fDst.Close()` to prevent leak. Without it, the file descriptors would remain open if the function exits through any error condition.

Reply:
No issue anymore since doesn't use copy operation.